### ISSUE TITLE
Banner: Close button should have a more distinct `aria-label`

### ISSUE THREAD

**Issue by eddysims** [GetJobber/atlantis#695]
Close button on the `Banner` should be more distinct `aria-label` (ex. "Close notification") so its clear what it relates to rather than just "close"

**Comment by crypto-cmd** [comment]
Hey @eddysims! I changed the aria-label to `Close this notification`. I will create a pull request when hacktober begins. :)
